app/javascript/components/public_health/custom_export/PatientsFilters.js

```diff
@@ -61,7 +61,7 @@ class PatientsFilters extends React.Component {
                   this.props.onQueryChange('tab', 'all');
                 }}
                 value={this.props.query?.workflow}>
-                <option value="all">All</option>
+                <option value="global">Global</option>
```

Comment:
we're going to need to include a migration or script to update the existing saved filters

Reply:
Ah good call. Will add that

Reply:
Done https://github.com/SaraAlert/SaraAlert/pull/937/commits/0b8b162f013e243cae2ec52b123a06d897fc5b25